example/wbs-precheck

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
I guess one of the files is there due to renaming hick ups, lets remove one of them. As we are not planning to install them in the system (do we?), I'd go with just `./precheck`. But `./wbs-precheck` would be fine with me too.

Reply:
@rti "hiccups" or antiquated "hiccoughs"

Reply:
Oh yes, that was entirely unconscious and that was the reason. This was `wbs-precheck` which I prefer as I have reasons to believe we should lean into this `wbs` acronym as a possible identity / rename for `example`. It's a same-paging discussion though so I was renaming to simply `precheck` to slim down the things to discuss here, but shot myself in the foot :) I would prefer to leave it `precheck` for now so will remove the extra file.